src/partitioned-dbs/index.rst

```diff
@@ -105,14 +105,20 @@ sensor readings from an IoT field monitoring service. In this particular
 use case, it's quite logical to group all documents by their ``sensor_id``
 field. In this case, we would call the ``sensor_id`` the partition.
 
+A partition offers two benefits. Firstly, it allows you to simplify the
+workload on the query engine, by allowing the engine to only query shards
+that it needs to query. Secondly, it allows the database to scale in more
+predictable ways.
+
```

Comment:
Several thoughts on this paragraph:

1. It's not technically accurate to say "A partition", since the whole point is multiple partitions.
2. I think there can be more than _only_ two benefits, so we shouldn't imply that these are the only benefits.
3. It also doesn't really "simplify" the workload, but rather reduces it (partitioning is inherently more complex, actually, at least in the mathematical sense)
4. "only query the shards it needs to query" doesn't seem clear to me. It sounds like a tautology.

I would suggest something like:

> Partitioning offers two primary benefits. First, it reduces the workload on the query engine by
> allowing it to query only a subset of shards. Second, it allows the database to scale in more 
> predictable ways.